play-app/src/Context/ProductCont.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { createContext,useContext,useState,useEffect } from "react";
+import useAxios from "../Utils/useAxios";
+
+const productContext = createContext()
+
+const ProductProvider = ({children}) =>{
+  const [prodData, setProdData] = useState([]);
+    const {apiData,loader} = useAxios("/api/products");
```

Comment:
Always use useEffect for Api calls.

Reply:
created a useAxios file in utils.which fetch data from backend using useEffect